skbio/stats/ordination/_principal_coordinate_analysis.py

```diff
@@ -142,3 +142,63 @@ def pcoa(distance_matrix):
                              columns=axis_labels),
         proportion_explained=pd.Series(proportion_explained,
                                        index=axis_labels))
+
+
+def pcoa_biplot(ordination, y):
+    """Compute the projection of descriptors into a PCoA matrix
+
+    This implementation is as described in Chapter 9 of Legendre & Legendre,
+    Numerical Ecology 3rd edition.
+
+    Parameters
+    ----------
+    ordination: OrdinationResults
+        The computed principal coordinates analysis of dimensions (n, c) where
+        the matrix ``y`` will be projected onto.
+    y: DataFrame
+        Samples by features table of dimensions (n, m). These can be
+        environmental features or abundance counts. This table should be
```

Comment:
Would it make much sense to feed in the same table used to calculate the PCoA?  Something seems a bit circular about using abundance counts ...

Reply:
Although circular, it seems fine. The book makes mentions of this use-case and it is the use-case of the biplots impelmentation in QIIME 1.

From a different book by Legendre:

> [On PCoA biplots] ... For projection of species data onto a PCoA plot, it is important to use the species data with the same transformation (if any) as the one used to compute the dissimilarity matrix.